sqlx-postgres/src/row.rs

```diff
@@ -48,3 +50,23 @@ impl ColumnIndex<PgRow> for &'_ str {
             .map(|v| *v)
     }
 }
+
+impl Debug for PgRow {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "PgRow: ")?;
```

Comment:
```suggestion
        write!(f, "PgRow ")?;
```

If you omit the colon then the debug output will look like a struct.